cfgov/unprocessed/apps/youth-employment-success/js/route-option-view.js

```diff
@@ -43,29 +44,35 @@ function RouteOptionFormView( element, { store } ) {
    * @param {object} updateObject object with DOM event and field name
    */
   function _setQuestionResponse( { name, event } ) {
-    const method = actionMap[name];
+    const action = actionMap[name];
+    const { target: { value }} = event;
```

Comment:
TRIVIAL: should be `const { target: { value } } = event;` looks like

Reply:
I wonder why the linter didn't catch this! 😠